client/app/scripts/controllers/ci_incident.js

```diff
@@ -8,7 +8,32 @@
  * Controller of the clientApp
  */
 angular.module('clientApp')
-  .controller('CiIncidentCtrl', function ($scope, incident) {
+  .controller('CiIncidentCtrl', function ($scope, incident, Builds) {
 
     $scope.incident = incident[0];
+
+    Builds.get($scope.incident.ci_build, 'ci_build').then(function(data) {
+      $scope.build = data[0];
+    });
+
+    var incidentStart = $scope.incident.created;
+    var incidentEnd = $scope.incident.created != $scope.incident.updated ? $scope.incident.updated : (new Date().getTime())/1000;
```

Comment:
incidentEnd should be determined by checking if there's a reference to `fixed_build`